### ISSUE TITLE
Need to use NwkKey as AppKey

### ISSUE THREAD

**Issue by jpmeijers** [SeongJiIoT/LSM100A_SDK#5]
At startup the LSM100A prints out its keys. On firmware V1.0.1 it looks like this:
```
[10:31:12:656] ------ OTAA ------␍␊
[10:31:12:677] >> AppKey:      12:34:56:78:12:34:56:78:12:34:56:78:12:34:56:78␍␊
[10:31:12:745] >> NwkKey:      87:65:43:21:87:65:43:21:87:65:43:21:87:65:43:21␍␊
```
(obvious hardcoded default values)

To make the OTAA Join work I had to use the NwkKey to register the device on the network server. When I used the AppKey the Join does not work.

@SeongJiIoT confirmed that this device uses LoRaWAN 1.0.2, so the expectation is that the AppKey should be used for the OTAA Join, as LoRaWAN v1.0.x does not have the concept of a NwkKey.

**Comment by SeongJiIoT** [comment]
Join operation is possible only when our module sets NwkKey and AppKey to be the same with AT command.
LoRa key data can be saved from FW 1.0.1 or later, so if you set it for the first time, the key data will not change even if the module is restarted.

**Comment by jpmeijers** [comment]
> Join operation is possible only when our module sets NwkKey and AppKey to be the same with AT command.

This is not true. I did not have to do this. I only needed to use the "NwkKey" as the AppKey when I registered my device. 

The problem is your LoRaWAN stack uses the v1.1.x naming conventions, but because you set it to do v1.0.2B, one need to translate it to the v1.0.x naming conventions when registering the device.

See https://github.com/TheThingsNetwork/lorawan-stack/issues/2600
> This can be interpreted as that AppKey was renamed to NwkKey, and a new AppKey has been introduced that is used for LoRaWAN 1.1 devices.
